types/hash_test.go

```diff
@@ -27,23 +29,23 @@ func TestCheckBigIntInField(t *testing.T) {
 }
 
 func TestNewHashAndBigIntFromBytes(t *testing.T) {
-	b := bytes.Repeat([]byte{1}, 32)
+	b := bytes.Repeat([]byte{1, 2}, 16)
```

Comment:
Wondering the reason of changing the testing data. What about just keeping both of them?

Reply:
New testcase makes sure that endianness is correctly handled. Keeping both of them is actually redundant, new testcase covers the old one as well.